load-code/pe/potential-shellcode-loader.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: potential shellcode loader
+    namespace: load-code/pe
+    author: "john.gorman@fireeye.com"
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    references:
+      - https://github.com/EgeBalci/amber
+    examples:
+      - bb7922d368a9a9c8d981837b5ad988f1
+  features:
+    - or:
+      - and:
+        - api: kernel32.LoadLibraryA
+        - api: kernel32.GetProcAddress
+        - api: kernel32.GetModuleHandleA
+        - api: kernel32.VirtualAlloc
```

Comment:
i suspect that this rule may match on too many functions, especially those unrelated to shellcode loading.

i'd propose that we prefer the existing rules in `load-code/pe` https://github.com/fireeye/capa-rules/tree/master/load-code/pe